static/locales/en-US.yaml

```diff
@@ -1007,6 +1008,9 @@ Tooltips:
     External Link Handling: |
       Choose the default behavior when a link, which cannot be opened in FreeTube, is clicked.
       By default FreeTube will open the clicked link in your default browser.
+    Open Deep Links In New Window: Choose whether URLs passed to FreeTube, such as by
```

Comment:
Wording makes me think that i can choose the desired behavior if i enable this setting. 

Change it to:

URLs passed to FreeTube, such as by redirect browser extensions or command line arguments, are opened in a new window.

Reply:
BTW, I wrote a redirect that doesn't require a browser extension: https://pakoito.github.io/freetube-forwarder/